src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookTriggerImpl.java

```diff
@@ -289,9 +289,11 @@ private boolean isMirrorConfigurationDifferent(BitbucketSCMRepository r) {
             return isEmpty(r.getMirrorName()) ^ isEmpty(r.getMirrorName());
         }
 
+        @SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH", justification = "None. No idea what this is about")
```

Comment:
still no idea? It would be good to put a more descriptive message here. Even if it is "investigated, deemed to be false positive"

Reply:
Figured it out- I  had missed a few @CheckForNull annotations on the code path. The end of each path can handle nulls so I don't think this is an issue.